packages/plugin-ext/src/plugin/tests.ts

```diff
@@ -483,8 +491,30 @@ export class TestRunProfile implements theia.TestRunProfile {
     @observableProperty('notifyPropertyChange')
     label: string;
 
-    @observableProperty('notifyPropertyChange')
-    isDefault: boolean;
+    _isDefault: boolean;
+
+    get isDefault(): boolean {
+        return this._isDefault;
+    }
+
+    set isDefault(isDefault: boolean) {
+        if(this.doSetDefault(isDefault)) {
+            this.proxy.$updateTestRunProfile(this.controllerId, this.profileId, { isDefault: isDefault });
+        }
+    }
+
+    doSetDefault(isDefault: boolean): boolean {
+        let change = false;
```

Comment:
Why not just 
```
if (this._isDefault !== isDefault) {
            this._isDefault = isDefault;
            this.onDidChangeDefaultEmitter.fire(isDefault);
            return false;
        }
        return false;

Reply:
Good one! Some remnants of old checkstyles rules :/ That will be fixed.